Resources/Prototypes/Entities/Structures/Wallmounts/intercom.yml

```diff
@@ -0,0 +1,170 @@
+- type: entity
+  id: Intercom
+  name: intercom
+  description: An intercom. For when the station just needs to know something.
+  components:
+  - type: WallMount
+  - type: ApcPowerReceiver
+  - type: Electrified
+    enabled: false
+    usesApcPower: true
+  - type: RadioMicrophone
+    listenRange: 2
```

Comment:
Wallmount should stop people from interacting with it from the other side of the wall, but iirc nothing would stop the range check from ignoring the walls, so people might still be able to do things like user a command intercom from maint tunnels or something.

So the radio mic should probably have a (data-field configurable) mechanic where it cancels `ListenAttemptEvent` if the speaker is not obstructed?